src/api/authentication.js

```diff
@@ -217,5 +225,42 @@ export async function authenticate (ctx, next) {
   return next()
 }
 
+export async function getEnabledAuthenticationTypes (ctx, next) {
+  // Log level for messages logged
+  let logLevel = 'error'
+
+  try {
+    if (!authorisation.inGroup('admin', ctx.authenticated)) {
+      ctx.statusCode = 403
+      const error = `User ${ctx.authenticated.email} is not an admin, API access to get enabled authentication types denied.`
+      logLevel = 'info'
+      throw Error(error)
```

Comment:
I know that `throw Error()` and `throw new Error()` are equivalent, but for project consistency could you please change this to `new Error()` (or if you really want, change all the existing `new Error()`s to `Error()`).
I do like the efficiency of just `Error()` but whatever you choose we need to keep one style throughout the repo

Reply:
addressed in https://github.com/jembi/openhim-core-js/pull/1082/commits/562dc5c4c469b99a04afcc0d09a9d002436aac15